cmake/libint2.pc.cmake.in

```diff
@@ -0,0 +1,10 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=@CMAKE_INSTALL_PREFIX@
+libdir=@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_LIBDIR@
+includedir=@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_INCLUDEDIR@
```

Comment:
For the record, cmake is really bad at this but it can be worked around: https://github.com/jtojnar/cmake-snips

(The existing autotools build handles this properly, because `@libdir@` `@includedir@` etc. work properly and support pkg-config's own runtime `${prefix}` expansion. CMake developers hate pkg-config because of NIH though.)

Reply:
Thanks, I think I've fixed this up at 1574ad2dd281e2272e87d08bae24c613f3900dd4 . Below is the diff for generated `libint2.pc` (with `-DCMAKE_INSTALL_LIBDIR=mylib -DCMAKE_INSTALL_INCLUDEDIR=myinclude`). Do you need a variable to shift the installed `.pc` location or is `${CMAKE_INSTALL_LIBDIR}/pkgconfig/` fine?

```
< libdir=/psi/gits/libint2-efv/build-22/library-install-stage/mylib
< includedir=/psi/gits/libint2-efv/build-22/library-install-stage/myinclude
---
> libdir=${exec_prefix}/mylib
> includedir=${prefix}/myinclude
```

Reply:
`${CMAKE_INSTALL_LIBDIR}/pkgconfig` is correct (especially since this is only calculated at install time, while pkg-config files are processed as runtime data).